lib/sles4sap/ipaddr2.pm

```diff
@@ -765,26 +766,43 @@ die in case of failure
 sub ipaddr2_os_connectivity_sanity {
     my (%args) = @_;
     $args{bastion_ip} //= ipaddr2_bastion_pubip();
+    my $ping_cmd = 'ping -c 3';
 
     # proceed_on_failure needed as ping or nc
```

Comment:
I think `proceed_on_failure` does not do anything with `script_run`. It does with `script_output`